lib/smart_answer_flows/calculate-agricultural-holiday-entitlement/calculate_agricultural_holiday_entitlement.erb

```diff
@@ -9,6 +9,6 @@
 <% govspeak_for :body do %>
   Work out how much statutory holiday leave you're entitled to as an agricultural worker.
 
-  Agricultural holiday entitlement is different in [Scotland](http://www.scotland.gov.uk/Publications/2012/10/7886/5){:rel="external"}
-  and [Northern Ireland.](http://www.dardni.gov.uk/index/fisheries-farming-and-food/enforcement-awb/awb-current-terms-and-conditions-of-employment-wages-etc.htm){:rel="external"}
+  Agricultural holiday entitlement is different in [Scotland](https://www.webarchive.org.uk/wayback/archive/20170105180821/http://www.gov.scot/Publications/2012/10/7886){:rel="external"}
```

Comment:
@richardskipper just wanted to double check this URL is the one we want to link to?

Reply:
It is, yes.

Reply:
@edwardkerry It is yes.